indico/modules/events/contributions/models/subcontributions.py

```diff
@@ -131,6 +131,14 @@ def timetable_entry(self):
         """Convenience property so all event entities have it"""
         return self.contribution.timetable_entry
 
+    @property
```

Comment:
It's true, the `Subcontribution ` properties are more or less a subset of the `Contribution ` ones, I would also go for a mixin